cross/openssl/digests.orig

```diff
@@ -0,0 +1,9 @@
+<<<<<<< HEAD
```

Comment:
This needs fixing. Don't `git pull` in a branch you use for PR's, unless you also rebase.